joltify/chain.json

```diff
@@ -0,0 +1,135 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "joltify",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.joltify.io/",
+  "pretty_name": "Joltify",
+  "chain_id": "joltify_1729-1",
+  "bech32_prefix": "jolt",
+  "daemon_name": "jolt",
+  "node_home": "$HOME/.joltify",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ujolt",
+        "low_gas_price": 0,
+        "average_gas_price": 0,
+        "high_gas_price": 0
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ujolt"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/joltify-finance/joltify_lending",
+    "genesis": {
+      "genesis_url": "https://ewr1.vultrobjects.com/joltify-mainnet/genesis_file/genesis.json"
+    },
+    "recommended_version": "v0.1.0-rc1",
+    "compatible_versions": [
+      "v0.1.0-rc1"
+    ],
+    "cosmos_sdk_version": "v0.47.5",
+    "consensus": {
+      "type": "cometbft",
+      "version": "v0.37.3"
+    },
+    "cosmwasm_version": "",
+    "cosmwasm_enabled": false,
+    "ibc_go_version": "v7.3.0",
+    "go_version": "1.22.3",
+    "binaries": {
+      "linux/amd64": "https://gitlab.com/api/v4/projects/42057472/packages/generic/joltify-linux-amd64-v0.1.0-rc1/v0.1.0-rc1/joltify"
+    },
+    "versions": [
+      {
+        "name": "v0.1.0-rc1",
+        "tag": "v0.1.0-rc1",
+        "height": 0,
+        "binaries": {
+          "darwin/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v3.1.0/osmosisd-3.1.0-darwin-amd64?checksum=sha256:a532f25ae754d2573f6a3c91ba59496ddb9f6766ccf6f69f408f6e1597144a74",
```

Comment:
I suspect these binaries were left over from copy and paste...

Reply:
Please copy the properties describing the version into this array as well (all the properties from recommended_version to binaries)